test/utils/image/manifest_test.go

```diff
@@ -156,7 +156,7 @@ func TestGetOriginalImageConfigs(t *testing.T) {
 
 func TestGetMappedImageConfigs(t *testing.T) {
 	originals := map[ImageID]Config{
-		10: {registry: "docker.io", name: "source/repo", version: "1.0"},
```

Comment:
why is this change?

Reply:
I see, is this one https://github.com/kubernetes/kubernetes/pull/132655#issuecomment-3242135145